x/concentrated-liquidity/fees.go

```diff
@@ -1,321 +0,0 @@
-package concentrated_liquidity
```

Comment:
It seems that this file ended up being "deleted" in Git history. Can you please revert it and, instead, rename it so that the full history is preserved?

Reply:
This seems to be because the file diff was large enough where it passed some threshold in git. Will revert changes to those files, rename first, then restore changes which should fix the issue.